packages/react-components/react-toolbar/stories/Toolbar/ToolbarDefault.stories.tsx

```diff
@@ -10,7 +10,7 @@ import { Toolbar, ToolbarButton, ToolbarDivider } from '@fluentui/react-componen
 import type { ToolbarProps } from '@fluentui/react-components/unstable';
 
 export const Default = (props: Partial<ToolbarProps>) => (
-  <Toolbar {...props}>
+  <Toolbar aria-label="Toolbar Default" {...props}>
```

Comment:
Please for tiny improvement. 
Don't include "toolbar" keyword into name because then screen reader repeats the "toolbar" as this is already narrated as role.
![image](https://user-images.githubusercontent.com/22620150/211494635-b84b4502-04c7-46c0-8584-a0d558a42375.png)
Please remove "Toolbar" string rom others examples, as well. Thank you.

Reply:
Done https://github.com/microsoft/fluentui/pull/26275/commits/e7f11bdcda043667af97d14a1d33accaf9de5fc9